lib/compress/zstd_compress.c

```diff
@@ -4760,12 +4818,17 @@ static size_t ZSTD_loadDictionaryContent(ZSTD_matchState_t* ms,
         ZSTD_fillHashTable(ms, iend, dtlm, tfp);
         break;
     case ZSTD_dfast:
+#ifndef ZSTD_EXCLUDE_DFAST_BLOCK_COMPRESSOR
```

Comment:
What happens in this case ?
It seems that the dictionary content gets silently skipped, and the user will just never get matches into the dictionary, but I suspect everything continues to work normally, so the impact can remain invisible.
Would it be better to return an error ?

Reply:
FIxed to assert that it's not called if excluded.